content/manuals/docker-hub/builds/link-source.md

```diff
@@ -140,6 +139,7 @@ unlink it both from Docker Hub, and revoke authorization in your Bitbucket accou
 3. From the **Linked accounts** tab, select the **Plug** icon next to the source provider you want to remove.
 
 > [!IMPORTANT]
+>
```

Comment:
Not necessary. In fact, I think the canonical format is without a space -- see the GitHub docs 

```suggestion
```